### ISSUE TITLE
Helpful install message

### ISSUE THREAD

**Issue by sandstrom** [rbenv/rbenv#1501]
It would be great if the "version x.y.z is not installed" message could be extended, explaining how to install the required version.

AFAIK running `rbenv install` should solve the problem.

I think this is the relevant source:

```sh
RBENV_PREFIX_PATH="${RBENV_ROOT}/versions/${RBENV_VERSION}"
if [ ! -d "$RBENV_PREFIX_PATH" ]; then
  echo "rbenv: version \`${RBENV_VERSION}' not installed" >&2
  exit 1
fi
```

https://github.com/rbenv/rbenv/blob/d604acb78aeba583be95f08d45eeae430372beb9/libexec/rbenv-prefix#L36-L40

## Possible solution

Perhaps something like this could work:

```sh
RBENV_PREFIX_PATH="${RBENV_ROOT}/versions/${RBENV_VERSION}"
if [ ! -d "$RBENV_PREFIX_PATH" ]; then
  echo "rbenv: version \`${RBENV_VERSION}' not installed" >&2
  echo "rbenv: you can try installing it with 'rbenv install'" >&2
  exit 1
fi
```

## Examples of this problem

This gist contains a lot of comments on this topic (to illustrate that I'm not the only one with this issue).

https://gist.github.com/esteedqueen/b605cdf78b0060299322033b6a60afc3

**Comment by jasonkarns** [comment]
Seems like it could be helpful; though it's worth considering that `rbenv install` may not be available. rbenv-install is provided by ruby-build when installed as an rbenv plugin. But ruby-build is optional and not required for rbenv to work. Which means the rbenv install command isn't guaranteed to exist.

**Comment by sandstrom** [comment]
> Seems like it could be helpful; though it's worth considering that `rbenv install` may not be available. rbenv-install is provided by ruby-build when installed as an rbenv plugin. But ruby-build is optional and not required for rbenv to work. Which means the rbenv install command isn't guaranteed to exist.

Good point! Should gate on that script being installed. Since it would be an additive message, it's not a problem if that wouldn't be shown to everyone.

**Comment by mislav** [comment]
I'm open to this! Thanks for the proposal and thanks @jasonkarns for pointing out the important gotcha.
